### PR TITLE
Creating module 'labels'

### PR DESCRIPTION
**Pull request by EmmaRenauld**
One step into the cleaning of the modules. It seems an easy one to start with, not too much chances to get mixed up with something else.

I placed it in `scilpy.image.labels.py`.

I cleaned the 5 labels scripts that I found. Their main now only do imports and saving.
- scil_combine_labels
- scil_dilate_labels
- scil_remove_labels
- scil_split_volume_by_ids
- scil_split_volume_by_labels

I moved `scil.io.image.get_data_as_label` into scil.image.labels, explaining the other modifications. It could also be kept in io as it does fetch the data from the img.

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by frheault** [comment]
Great ! When 1.4 is released, and this PR is updated with master (just in case) I will test to check if the data before/after is the same.

I think all these functions could be tested with a fake 4x4 array, when I test it I will try to generate some data for unit tests.

File: scilpy/image/labels.py [hunk 1]
```diff
@@ -0,0 +1,281 @@
+# -*- coding: utf-8 -*-
+import inspect
+import logging
+import os
+
+import numpy as np
+from scipy.spatial.ckdtree import cKDTree
+
+import scilpy  # ToDo. Is this the only way?
+
```
Comment (EmmaRenauld) [inline scilpy/image/labels.py:10]: To put line 48

File: scilpy/image/labels.py [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+def get_lut_dir():
+    """
+    Return LUT directory in scilpy repository
+
+    Returns
+    -------
+    lut_dir: string
+        LUT path
+    """
+    # Get the valid LUT choices.
```
Comment (EmmaRenauld) [inline scilpy/image/labels.py:47]: Not the best code. Will add an issue for eventually improving.

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

File: scripts/scil_split_volume_by_ids.py [hunk 3]
```diff
@@ -89,31 +63,25 @@ def main():
...
-    cnt_filename = 0
-    for label in label_indices:
-        if int(label) != 0:
-            split_label = np.zeros(label_img.shape,
-                                   dtype=np.uint16)
-            split_label[np.where(label_img_data == int(label))] = label
+    split_data = split_labels(label_img_data, label_indices)
 
-            split_image = nib.Nifti1Image(split_label,
+    for i in range(len(label_indices)):
```
Comment (frheault) [inline scripts/scil_split_volume_by_ids.py:79]: This list is always out of range singe label_indices include the 0 index, but the filenames excludes it.

Right now labels 2 is saved as PREFIX_1.nii.gz (shifted)

**Review by frheault — APPROVED** [review]
I think this works well (except for one thing), if there is another correction I will do it in the next PR

(Immediately after this one I plan on doing unit tests for all new functions (replacing `main`)

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

File: scripts/scil_split_volume_by_ids.py (hunk 3 above)
Comment (EmmaRenauld) [reply scripts/scil_split_volume_by_ids.py:80]: I'm not sure that this was the issue. I think I fixed it nonetheless. Can you verify?

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

File: scripts/scil_split_volume_by_ids.py (hunk 3 above)
Comment (EmmaRenauld) [reply scripts/scil_split_volume_by_ids.py:84]: Fixed.

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Pull request merged**